### PR TITLE
Remove deprecated /deep/ selectors, update polymer elements and fix spelling mistake

### PR DESCRIPTION
**Pull request by KaptenJansson**
**Description**
Removes the use of /deep/ selectors
Update polymer elements
Fix cron.yml spelling mistake

**Purpose**
/deep/ is deprecated in Chrome m60 and breaks the app.

### CODE REVIEW

**Comment by KaptenJansson** [comment]
Fixes issue reported in both [chromium](https://bugs.chromium.org/p/chromium/issues/detail?id=739263) and[ discuss-webrtc](https://groups.google.com/forum/#!topic/discuss-webrtc/-QuHsTvnwvA).

File: cron.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+cron:
+- description: Cleanup blobstore files.
```
Comment (MirkoBonadei) [inline cron.yaml]: Is the - a typo?

Comment (KaptenJansson) [inline cron.yaml]: Nope https://cloud.google.com/appengine/docs/standard/python/config/cronref

**Comment by KaptenJansson** [comment]
I did however for some reason create a new app.yaml file rather than renaming it. Fixed that. Sloppy of me.....

**Pull request merged**